config/emails_pl.py

```diff
@@ -259,6 +259,24 @@ class EmailPytrening(EmailData):
 WAŻNA INFROMACJA! Mamy tylko 200 miejsc na sali, a rejestracja jest obowiązkowa ze względu na przepisy BHP.
     """
 
+class EmailFeedback(EmailData):
+    recipients = {'accepted': True, 'confirmation': 'ack', 'mentor': False, 'organiser': False}
```

Comment:
tak jak sugerowalem wcześniej do wszystkich tzn: bez tej 'confirmation': 'ack',
ale to porpawie sam